plugins/inputs/gnmi/gnmi_test.go

```diff
@@ -424,10 +424,10 @@ func TestSubscribeResponseError(t *testing.T) {
 	ml := &MockLogger{}
 	plugin := &GNMI{Log: ml}
 	// TODO: FIX SA1019: gnmi.Error is deprecated: Do not use.
-	errorResponse := &gnmi.SubscribeResponse_Error{Error: &gnmi.Error{Message: me, Code: mc}}
-	plugin.handleSubscribeResponse("127.0.0.1:0", &gnmi.SubscribeResponse{Response: errorResponse})
+	errorResponse := &gnmiLib.SubscribeResponse_Error{Error: &gnmiLib.Error{Message: me, Code: mc}}
```

Comment:
up to you on whether you want to address this deprecation notice

Reply:
@ssoroka we agreed with @srebhan that it should be handled by someone with better `gnmi` knowledge we leave it as it is for now.